JAICore/jaicore-basic/src/main/java/jaicore/basic/sets/LDSRelationComputer.java

```diff
@@ -36,51 +36,52 @@
 		int indexOfValue;
 
 		public Node() {
-			
+
 		}
-		public Node(Node parent, int indexOfSet, int defficiency, int indexInSet) {
+		public Node(final Node parent, final int indexOfSet, final int defficiency, final int indexInSet) {
 			long start = System.currentTimeMillis();
 			this.parent = parent;
 			this.indexOfSet = indexOfSet;
 			this.defficiency = defficiency;
 			this.indexOfValue = indexInSet;
 			assert System.currentTimeMillis() - start <= 1 : "Constructor execution took more than 1ms: " + (System.currentTimeMillis() - start) + "ms";
 		}
-		
-		public void fillTupleArrayWithValues(List<T> tupleArray) {
+
+		public void fillTupleArrayWithValues(final List<T> tupleArray) {
 			tupleArray.clear();
-			fillTupleArrayWithValuesRec(tupleArray);
+			this.fillTupleArrayWithValuesRec(tupleArray);
 		}
-		
-		private void fillTupleArrayWithValuesRec(List<T> tupleArray) {
-			if (parent == null)
+
+		private void fillTupleArrayWithValuesRec(final List<T> tupleArray) {
+			if (this.parent == null) {
 				return;
-			tupleArray.add(0, getObject());
-			parent.fillTupleArrayWithValuesRec(tupleArray);
+			}
+			tupleArray.add(0, this.getObject());
+			this.parent.fillTupleArrayWithValuesRec(tupleArray);
 		}
 
 		@Override
 		public String toString() {

```

Comment:
why not JSON here?

Reply:
This is an internal class. The toString method is actually not even ever used. I just removed it.